Fix installation path of distutils on Gentoo 64bits (installing to lib64...)

--- cmake/templates/python_distutils_install.sh.in~	2014-07-11 10:36:13.000000000 +0200
+++ cmake/templates/python_distutils_install.sh.in	2015-01-26 15:37:33.978602077 +0100
@@ -25,4 +25,4 @@
     build --build-base "@CMAKE_CURRENT_BINARY_DIR@" \
     install \
     $DESTDIR_ARG \
-    @SETUPTOOLS_ARG_EXTRA@ --prefix="@CMAKE_INSTALL_PREFIX@" --install-scripts="@CMAKE_INSTALL_PREFIX@/@CATKIN_GLOBAL_BIN_DESTINATION@"
+    @SETUPTOOLS_ARG_EXTRA@ --prefix="@CMAKE_INSTALL_PREFIX@" --install-scripts="@CMAKE_INSTALL_PREFIX@/@CATKIN_GLOBAL_BIN_DESTINATION@" --install-lib="@CMAKE_INSTALL_PREFIX@/@PYTHON_INSTALL_DIR@"
